checksum.sh

```diff
@@ -13,7 +13,7 @@ checksum_file() {
 FILES=()
 while read -r -d ''; do
 	FILES+=("$REPLY")
-done < <(find . -type f \( -name "build.gradle*" -o -name "gradle-wrapper.properties" \) -print0)
+done < <(find . -type f \( -name "build.gradle*" -o -name "Configs.kt" -o -name "Dependencies.kt" -o -name "gradle-wrapper.properties" \) -print0)
```

Comment:
In fact, `Configs` file doesn't impact on cache size at all. I think we can remove `Configs` and revert this to one previous version.

Reply:
Done. 52053c9